src/components/content/deployedServices/myServices/MyServiceHistory.tsx

```diff
@@ -0,0 +1,115 @@
+
+export const MyServiceHistory = ({
+    deployedService,
+}: {
+    deployedService: DeployedServiceDetails | VendorHostedDeployedServiceDetails;
+}): React.JSX.Element => {
+    let serviceModificationAuditHistoryList: ServiceModificationAuditDetails[] = [];
+    const listServiceModifyHistoryQuery = useListServiceModifyHistoryQuery(deployedService.id);
+
+    if (listServiceModifyHistoryQuery.isSuccess && listServiceModifyHistoryQuery.data.length > 0) {
+        serviceModificationAuditHistoryList = listServiceModifyHistoryQuery.data;
+    }
+
+    const columns: ColumnsType<ServiceModificationAuditDetails> = [
+        {
+            title: 'ModifyId',
+            dataIndex: 'id',
+            align: 'center',
+            width: 100,
+        },
+        {
+            title: 'Previous',
+            dataIndex: 'previousDeployRequest',
+            width: 300,
+            render: (record: DeployRequest) => {
+                return (
+                    <ul style={{ whiteSpace: 'normal', wordBreak: 'break-word' }}>
+                        <li>
+                            <Text strong>customerServiceName:</Text>&nbsp;{record.customerServiceName}
+                        </li>
+                        {record.serviceRequestProperties ? (
+                            <li>
+                                <Text strong>Service Request Properties:</Text>&nbsp;
+                                {JSON.stringify(record.serviceRequestProperties)}
+                            </li>
+                        ) : (
+                            <></>
+                        )}
+                    </ul>
+                );
+            },
+        },
+        {
+            title: 'New',
+            dataIndex: 'newDeployRequest',
+            width: 300,
+            render: (record: DeployRequest) => {
+                return (
+                    <ul style={{ whiteSpace: 'normal', wordBreak: 'break-word' }}>
+                        <li>
+                            <Text strong>customerServiceName:</Text>&nbsp;{record.customerServiceName}
+                        </li>
+                        {record.serviceRequestProperties ? (
+                            <li>
+                                <Text strong>Service Request Properties:</Text>&nbsp;
+                                {JSON.stringify(record.serviceRequestProperties)}
+                            </li>
+                        ) : (
+                            <></>
+                        )}
+                    </ul>
+                );
+            },
+        },
+        {
+            title: 'Status',
+            dataIndex: 'taskStatus',
+            align: 'center',
+            width: 50,
+            render: (value, record) => {
+                if (value === ServiceModificationAuditDetails.taskStatus.FAILED) {
+                    return (
+                        <Tooltip overlayInnerStyle={{ width: 400 }} placement='bottomRight' title={record.errorMsg}>
```

Comment:
is it possible to add some kind of ℹ️ icon next to the the tag so that the user knows that there is hidden information available when they click on it?